ftp/src/main/java/ch/cyberduck/core/ftp/FTPAttributesFinderFeature.java

```diff
@@ -56,18 +58,21 @@ public PathAttributes find(final Path file, final ListProgressListener listener)
         }
         try {
             try {
-                if(session.getClient().hasFeature(FTPCmd.MLST.getCommand())) {
-                    if(!FTPReply.isPositiveCompletion(session.getClient().sendCommand(FTPCmd.MLST, file.getAbsolute()))) {
-                        throw new FTPException(session.getClient().getReplyCode(), session.getClient().getReplyString());
-                    }
-                    final FTPDataResponseReader reader = new FTPMlsdListResponseReader();
-                    final AttributedList<Path> attributes
-                            = reader.read(file.getParent(), Arrays.asList(session.getClient().getReplyStrings()), listener);
-                    if(attributes.contains(file)) {
-                        return attributes.get(attributes.indexOf(file)).attributes();
+                final PreferencesReader preferences = new HostPreferences(session.getHost());
+                if(preferences.getBoolean("ftp.command.mlsd")) {
```

Comment:
Explicit property to avoid confusion `mlsd` vs. `mslt`?

Reply:
Thus just keep it as is.